File: Algoritimos-fundamentais-Entregavel1/Somatorio.java
import java.util.Scanner;

public class Somatorio {
    public static void main(String[] args) {
        Scanner teclado = new Scanner(System.in);

        System.out.print("Informe a quantidade de numeros: ");
        int n = teclado.nextInt();
        int soma = 0;

        for (int i = 0; i < n; i++) {
            System.out.print("Digite um numero: ");
            soma += teclado.nextInt();
        }

        System.out.printf("Somatorio = %d%n", soma);
        teclado.close();
    }
}

File: Algoritimos-fundamentais-Entregavel1/Somatorio.js
const readline = require("readline");

const entrada = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

let soma = 0;
let i = 0;

entrada.question("Informe a quantidade de numeros: ", function(n) {

    n = parseInt(n);

    function pedirNumero() {

        if (i < n) {

            entrada.question("Digite um numero: ", function(numero) {

                numero = parseInt(numero);

                soma += numero;
                i++;

                pedirNumero();
            });

        } else {

            console.log("Somatorio: " + soma);
            entrada.close();
        }
    }

    pedirNumero();
});